core/build.gradle

```diff
@@ -94,8 +94,8 @@ dependencies {
       exclude group: 'org.elasticsearch', module: 'elasticsearch'
     }
   }
-  testCompile 'com.google.jimfs:jimfs:1.1'
-  testCompile 'com.google.guava:guava:18.0'
+  testCompile "com.google.jimfs:jimfs:${versions.jimfs}"
+  testCompile "com.google.guava:guava:${versions.guava}"
```

Comment:
I'm not comfortable with this change for Guava. Guava is a common transitive third-party dependency and there are breaking differences between releases so I don't think we should unify these transitive dependencies around a single version.